test/tests/interfacekernels/1d_interface/no-failed-point-inversions.i

```diff
@@ -0,0 +1,139 @@
+[Mesh]
```

Comment:
You've got some mismatched hit styles here - make it consistent?

Reply:
👍